README.md

```diff
@@ -330,10 +369,85 @@ css file that styles the "pygmentized" code examples. We use
 blocks
 [here](https://github.com/trulia/hologram-example/tree/gh-pages/hologram_assets/doc_assets/css).
 
+### Custom Code Example Renderers
+
+By default, hologram supports the following code example types:
+
+- `html_example` and `html_example_table`
+- `haml_example` and `haml_example_table`
+- `slim_example` and `slim_example_table`
+- `js_example`
+- `jsx_example`
+
+Let's say you want to include coffeescript examples in your styleguide.
+You'll need to create a custom renderer for this.
+
+You can generate custom examples with the code example renderer factory:
+
+```ruby
+require 'coffee-script'
+
+Hologram::CodeExampleRenderer::Factory.define('coffee') do
+  example_template 'js_example_template'
+  table_template 'js_table_template'
+
+  lexer { Rouge::Lexer.find(:coffee) }
+
+  rendered_example do |code|
+    CoffeeScript.compile(code)
+  end
+end
+```
+
+Place this code in a ruby file in your code example renderers folder
+(i.e. `./code_example_renderers/coffee_renderer.rb`).
+Now you should be able to render coffeescript examples in our styleguide.
+
+We can render single coffeescript examples...
+
+	```coffee_example
```

Comment:
Should we add in files to support coffee so that we can show them the real files at the end of the docs? I know we don't need coffee support, but I think most people (when they aren't familiar with a code base) find something similar to what they are trying to create, copy it, and modify it.